gen_cert.py

```diff
@@ -652,3 +678,28 @@ def _generate_certificate(
             )
 
         return (download_uuid, verify_uuid, download_url)
+
+    @classmethod
+    def reuse_cert(cls, last_cert, new_course, new_designation):
```

Comment:
I like this thought, but I'd like it more if implemented like I mentioned in certificate_agent.py @ line ~100:

```python
if cert.is_reusable(course_id, designation):
    cert = None;
else:
    cert = CertificateGen(...)
```